web-ui/.prettierrc

```diff
@@ -0,0 +1,6 @@
+{
+  "arrowParens": "avoid",
+  "bracketSpacing": false,
+  "singleQuote": true,
```

Comment:
Sometimes single vs double quotes are necessary between operating systems is my understanding. Maybe best to not auto-adjust this one?

Reply:
Prettier handles that case. It knows to surround strings that contain single quotes with double and vice versa.